example_rails7/app/javascript/controllers/index.js

```diff
@@ -0,0 +1,8 @@
+// This file is auto-generated by ./bin/rails stimulus:manifest:update
+// Run that command whenever you add a new controller or create them with
+// ./bin/rails generate stimulus controllerName
+
+import { application } from "./application"
```

Comment:
```suggestion
import { application } from "./application.js"
```

Reply:
I think, if these changes are required, then our generator may need to change, since this code is generated by Rails.

Reply:
@OutlawAndy Yes, that's why I originally included those files in the webpack generator but I see you removed them in https://github.com/RoleModel/rolemodel_rails/pull/91/commits/718e998616f099779fbfc5925e839ba237c05f8e

Reply:
@BenEinwechter - Ya, that came from a conversation w/ @calebwoods about reducing surface area and attempting to inject changes rather than replace files, so generators can be used on existing projects and not just new ones..

If every import statement that Rails generates will need to be modified in order to work with our JS strategy, we may need to rethink our JS strategy..  Maybe we could pair early next week to figure out how we move forward with this?

I don't want to merge something that doesn't work, but I'd rather merge an incremental improvement, than leave this branch open for several more weeks.

Reply:
@BenEinwechter Is the issue just that webpack requires import statements to have file extensions?  If so, we could have a generator method that scans javascript files for import statements that don't have file extensions and assume that all they need is `.js` added to the path.  That would allow the generator to work with existing applications.  Even if there are edge cases where `.js` is not correct, the end user should be able to recognize these as they review generated changes..  Thoughts @calebwoods ?